app/models/ingredients_alchemical_property.rb

```diff
@@ -36,15 +36,9 @@ class IngredientsAlchemicalProperty < ApplicationRecord
   DOES_NOT_MATCH = 'is not consistent with any ingredient that exists in Skyrim'
 
   def canonical_models
-    matching_attrs = {
-      alchemical_property_id:,
-      ingredient_id: canonical_ingredients.ids,
-      strength_modifier:,
-      duration_modifier:,
-      priority:,
-    }.compact
+    return Canonical::IngredientsAlchemicalProperty.where(**attributes_to_match) if attributes_to_match.any?
```

Comment:
This is to create a more uniform API with other canonical models to prepare to generalise these into a single parent class. Not strictly related to this card.